CMakeLists.txt

```diff
@@ -301,10 +301,10 @@ endif()
 # ${CMAKE_INSTALL_PREFIX}/
 install(
   TARGETS executorch
-  DESTINATION lib
+  LIBRARY DESTINATION ${CMAKE_INSTALL_LIBDIR}
```

Comment:
Does CMake provide a default value for `CMAKE_INSTALL_LIBDIR` or we need to always pass in a value

Reply:
Yeah it defaults to `lib` but can be `lib64` in some situations, or a big ProgramFiles path on windows. So this is more portable then just saying `lib`.